### DISCUSSION TITLE
Deep Linking

### DISCUSSION THREAD

**Discussion by mauryalamin** [NuPlay/RichText#25] [Q&A] [answered]
Amazing library!

I'm trying to enable deep linking in my app, so that when a user taps the link (`<a href="my-app://detail/id">Deep Link</a>`) in the RichTextView, it opens another view within the app. This works when I use a UIViewRepresentable to render the HTML, but it crashes when using RichText. Is this currently possible with this library?

Best!

**Comment by NuPlay** [comment, marked as answer]
Sorry for replying so late 😓
I fixed that problem in v2.1.3
#28

**Reply by mauryalamin** [reply]
Excellent! Thanks
